gosu/pom.xml

```diff
@@ -53,10 +53,5 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
```

Comment:
mockito-all packages an older version hamcrest which conflicts with the one provided by junit and should not be used.